utest/reporting/test_jsmodelbuilders.py

```diff
@@ -19,7 +19,10 @@
 
 def remap(model, strings):
     if isinstance(model, StringIndex):
-        return strings[model][1:]
+        if strings[model].startswith('*'):
+            # Strip the asterisk from a raw string.
+            return strings[model][1:]
+        return strings[model]
```

Comment:
It would be, I agree. It was actually my first thought, but I couldn't find a decompress method in any of the libraries. I would have to write an inverse operation of what StringCache does. I'm totally up for doing that. Would you prefer that?